exec/java-exec/src/main/java/org/apache/drill/exec/store/SchemaTreeProvider.java

```diff
@@ -154,17 +156,23 @@ public SchemaPlus createFullRootSchema(SchemaConfig schemaConfig) {
           .build(logger);
     }
   }
+  */
 
   @Override
   public void close() throws Exception {
+    //TODO jnturton: clean up
+    /*
     List<AutoCloseable> toClose = Lists.newArrayList();
     for(SchemaPlus tree : schemaTreesToClose) {
       addSchemasToCloseList(tree, toClose);
     }
 
     AutoCloseables.close(toClose);
+    */
   }
 
+  //TODO jnturton: clean up
+  /*
   private static void addSchemasToCloseList(final SchemaPlus tree, final List<AutoCloseable> toClose) {
     for(String subSchemaName : tree.getSubSchemaNames()) {
       addSchemasToCloseList(tree.getSubSchema(subSchemaName), toClose);
```

Comment:
@vvysotskyi under this PR, the construction of the schema tree becomes lazy since createFullRootSchema is avoided.  The code at this line unfortunately undoes the benefits by eagerly constructing the entire schema tree only in order to be able to close schema objects.  I am not happy about skipping calls to `close()` on schemas, do you think I should look to add code to DynamicRootSchemas that returns only schemas that have already been constructed and initialised so that the code here has a suitable collection to close?  E.g. the `getSubSchemaNames()`, which returns every storage config name, gets replaced with something like `getInitialisedSubSchemas()`?

Reply:
Yes, initially I had doubts about this change, but for now, I propose to delete this method. For the case when someone would need to add some special logic under it, this change might be reconsidered. But we should be sure that changes in this PR are covered by tests, so breaking this optimization would be easily detected.